cvat-ui/src/components/task-page/validation-report.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import {
+    Row,
+    Col,
+    Button,
+    Spin,
+} from 'antd';
+import Text from 'antd/lib/typography/Text';
+
+interface Props {
+    taskInstance: any;
+}
+
+function ValidationReportComponent(props: Props): JSX.Element {
+    const [loading, setLoading] = useState(false);
+    const [report, setReport] = useState(null);
+    const { taskInstance } = props;
+
+    const loadData = () : void => {
+      setLoading(true);
+      taskInstance.validate().then(responseData => {
+          setReport(responseData.report);
+          setLoading(false);
+      })
+    };
+
+    if (loading) {
+        return (<Spin size='large' className='cvat-spinner' />);
+    }
+
+    let actionButton = report ?
```

Comment:
don't use tertiary operator for such long and complex statements, it becomes difficult to read
use normal if/else instead.